### PR TITLE
Added logging MQTT clientid on connection and publish refactoring

### PR DESCRIPTION
**Pull request by ppatierno**
Trivial PR to print the MQTT clientid on connection and a little refactoring around handling the publish message.

### CODE REVIEW

File: src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java [hunk 1]
```diff
@@ -131,6 +132,7 @@ private static byte[] payloadToBytes(MqttPublishMessage msg) {
      * @param publishMessage represents a MqttPublishMessage
      */
     private void handlePublishMessage(ChannelHandlerContext ctx, MqttPublishMessage publishMessage) {
+        publishMessage.retain();
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java:135]: Yep I was thinking the same but then I just moved into the handle what you had outside. Anyway going to remove them ;-)

File: src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java [hunk 2]
```diff
@@ -169,5 +171,6 @@ private void handlePublishMessage(ChannelHandlerContext ctx, MqttPublishMessage
             }
             case EXACTLY_ONCE -> logger.warn("QoS level EXACTLY_ONCE is not supported");
         }
+        publishMessage.release();
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java]: Yep, I addressed the comment.

**Review by antonio-pedro99 — APPROVED** [review]
LGTM

**Review by antonio-pedro99 — APPROVED** [review]
LGTM

**Review by kyguy — APPROVED** [review]

**Pull request merged**